sources_test.go

```diff
@@ -155,6 +155,40 @@ func TestListFilesGlob(t *testing.T) {
 	}
 }
 
+func TestFindFilesMultipleFiles(t *testing.T) {
+	t.Parallel()
+	dir := "testdata/multiple_files"
+	want := fmt.Sprintf("%s/1.txt\n%s/2.txt\n%s/3.tar.zip\n", dir, dir, dir)
+	got, err := ListFiles(dir).String()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if got != want {
+		t.Errorf("Want %q, got %q", want, got)
+	}
+}
+
+func TestFindFilesMultipleFilesWithSubdirectory(t *testing.T) {
+	t.Parallel()
+	dir := "testdata/multiple_files_with_subdirectory"
+	want := fmt.Sprintf("%s/1.txt\n%s/2.txt\n%s/3.tar.zip\n%s/dir/1.txt\n%s/dir/2.txt\n", dir, dir, dir, dir, dir)
```

Comment:
Can we add a 'hidden' file (one whose name starts with a dot) to these test cases? FindFiles should find it.